Resources/doc/annotations/routing.rst

```diff
@@ -1,190 +1,61 @@
 @Route and @Method
 ==================
 
-.. caution::
+**Routing annotations of the SensioFrameworkExtraBundle are deprecated** since
+version 5.2 because they are now a core feature of Symfony.
 
-    Routing annotations of the SensioFrameworkExtraBundle are deprecated since version 5.2.
-    Configuring routes with annotations is now a core feature of Symfony. Please use it
-    instead as explained in :doc:`/routing`.
+How to Update your Applications
+-------------------------------
 
-Usage
------
+``@Route`` Annotation
+~~~~~~~~~~~~~~~~~~~~~
 
-The ``@Route`` annotation maps a route pattern with a controller::
+The Symfony annotation has the same options as the SensioFrameworkExtraBundle
```

Comment:
The `service` option is not part of the core `@Route` annotation. Maybe it's worth to mention here that you should use FQCN as service ids now.